fast64_internal/oot/oot_anim.py

```diff
@@ -34,11 +34,12 @@
 
 
 class OOTAnimExportSettingsProperty(bpy.types.PropertyGroup):
+    isCustomFilename: bpy.props.BoolProperty(name="Use Custom Filename")
+    filename: bpy.props.StringProperty(name="Filename")
     isCustom: bpy.props.BoolProperty(name="Use Custom Path")
     customPath: bpy.props.StringProperty(name="Folder", subtype="FILE_PATH")
     folderName: bpy.props.StringProperty(name="Animation Folder", default="object_geldb")
```

Comment:
I've tried reviewing before but I just don't understand how filename / customPath / folderName play together, there may be something to improve here

To give a more concrete suggestion, could the "is custom" bool properties have a description with the behavior difference between not-custom and custom?

Reply:
This PR changes things so that the filename of an export by default will be based off the object/skeleton/animation name in blender. `filename` is used to provide a custom filename option to override that, if the user wants. `customPath` is for exporting to any non-decomp directory.